main/lsp/notifications/did_change_configuration.cc

```diff
@@ -0,0 +1,22 @@
+#include "main/lsp/notifications/did_change_configuration.h"
+#include "main/lsp/LSPPreprocessor.h"
+
+namespace sorbet::realmain::lsp {
+DidChangeConfigurationTask::DidChangeConfigurationTask(const LSPConfiguration &config,
+                                                       std::unique_ptr<DidChangeConfigurationParams> params)
+    : LSPTask(config, LSPMethod::WorkspaceDidChangeConfiguration), params(move(params)) {}
+
+LSPTask::Phase DidChangeConfigurationTask::finalPhase() const {
+    return LSPTask::Phase::RUN;
+}
+
+void DidChangeConfigurationTask::preprocess(LSPPreprocessor &preprocessor) {
```

Comment:
like the index comment, i believe we want to use this callback to update the preprocessor's globalstate(s) too

Reply:
Does it have one? (I don't see it)

Reply:
I believe you're correct that it doesn't have one. Thanks for checking!